linera-base/src/identifiers.rs

```diff
@@ -574,6 +584,83 @@ impl std::str::FromStr for Owner {
     }
 }
 
+impl Serialize for AccountOwner {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        match self {
+            AccountOwner::User(owner) => {
+                let key = format!("User:{}", owner);
+                serializer.serialize_str(&key)
+            }
+            AccountOwner::Application(app_id) => {
+                let key = format!("Application:{}", app_id);
+                serializer.serialize_str(&key)
+            }
+        }
```

Comment:
It feels like this serialization format should only used if `serializer.is_human_readable()`.